docs/products/flink/howto/connect-opensearch.rst

```diff
@@ -0,0 +1,56 @@
+Create an OpenSearch®-based Apache Flink® table
+===============================================
+
+To build data pipelines, Apache Flink® requires source and target data structures to `be mapped as Flink tables <https://ci.apache.org/projects/flink/flink-docs-release-1.13/docs/dev/table/sql/create/#create-table>`_. This functionality can be achieved via the `Aiven console <https://console.aiven.io/>`_ or :doc:`Aiven CLI </docs/tools/cli/service/flink>`.
+
+A Flink table can be defined over an existing or new Aiven for OpenSearch® index to be able to sink streaming data. To define a table over an OpenSearch® index, the index name and column data formats need to be defined, together with the Flink table name to use as reference when building data pipelines.
+
+.. Warning:: 
+
+    Aiven for OpenSearch® can only be used as the **target** of a data pipeline. You'll be able to create jobs that write data to an OpenSearch® index. Reading data from an OpenSearch® index is currently not possible.
+
+
+Create an OpenSearch®-based Apache Flink® table with Aiven Console
+------------------------------------------------------------------
+
+To create an Apache Flink table based on an Aiven for OpenSearch® index via Aiven console:
+
+1. Navigate to the Aiven for Apache Flink® service page, and open the **Jobs and Data** tab.
+
+   .. Warning::
```

Comment:
I've moved the Warning note into the body of this list item - I think it will look better.